minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Server/ServerStateMachine.java

```diff
@@ -807,6 +804,16 @@ private EntityPlayerMP getPlayerFromUsername(String username)
             return player;
         }
 
+        private void setMaxHealth(EntityLivingBase entity, float maxHealth) {
+            UUID MODIFIER_ID = UUID.fromString("d7efc338-70d7-42dd-afc0-b198a67c926e");
```

Comment:
Will do, I think I used all caps because it's a constant and that's usually how I mark that in python, but I can changed that. The magic string is just a hardcoded random string since this thing needs a unique UUID per entity modifier for a given entity.

Reply:
Right, it is a local constant, got it. I guess I personally only use all-caps for global constants.